tests/test_group.py

```diff
@@ -1007,7 +1007,7 @@ async def test_asyncgroup_create_array(
     assert subnode.dtype == dtype
     # todo: fix the type annotation of array.metadata.chunk_grid so that we get some autocomplete
     # here.
-    assert subnode.metadata.chunk_grid.chunk_shape == chunk_shape
+    assert subnode.chunk_grid.chunk_shape == chunk_shape
```

Comment:
Is this change related to library changes in this PR?

Reply:
Can we do a deprecation of that in a separate PR? It seems unrelated to data type changes.

Reply:
yep!

Reply:
this change has been taken out of this PR